Apsitvarkom.UnitTests/Models/Mapping/CleaningEventMappingTests.cs

```diff
@@ -58,5 +58,33 @@ public void CleaningEventToCleaningEventResponse()
             Assert.That(response.PollutedLocationId, Is.EqualTo(businessLogicObject.PollutedLocationId));
         });
     }
+
+    [Test]
+    public void CleaningEventCreateRequestToCleaningEvent()
```

Comment:
you put it in the `Response mappings` region, while this is a request -> business object mapping.